src/System.Collections.NonGeneric/tests/Helpers.cs

```diff
@@ -3,11 +3,14 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Globalization;
+using System.Runtime.InteropServices;
 
 namespace System.Collections.Tests
 {
     internal static class Helpers
     {
+        public static bool IsDesktopJob = RuntimeInformation.FrameworkDescription.Contains(".NET Framework");
```

Comment:
This isn't the best way to do this check. We need the following fixed https://github.com/dotnet/buildtools/issues/960, and instead should use the TFM.